ipaserver/plugins/cert.py

```diff
@@ -623,6 +626,7 @@ def execute(self, csr, all=False, raw=False, **kw):
         if not raw:
             self.obj._parse(result)
             result['request_id'] = int(result['request_id'])
+            result['cacn'] = ca_obj['cacn'][0]
```

Comment:
This should be `ca_obj['cn'][0]`, but we already the cn in variable `ca`
anyway - there is no need to get it from the `ca_obj`.